<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statusWidgetClass</class>
 <widget class="QWidget" name="statusWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/icons/resource/green.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="statusToolButton" name="toolButton_userName">
         <property name="text">
          <string>toolbutton</string>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::DownArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="statusTextWidget" name="statusTextWidgetObject" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="statusAvatarWidget" name="pushButton_avatar">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/resource/avatar.png</normaloff>:/icons/resource/avatar.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionBusy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resource/red.png</normaloff>:/icons/resource/red.png</iconset>
   </property>
   <property name="text">
    <string>Busy</string>
   </property>
  </action>
  <action name="actionAvailable">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resource/green.png</normaloff>:/icons/resource/green.png</iconset>
   </property>
   <property name="text">
    <string>Available</string>
   </property>
  </action>
  <action name="actionInvisible">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resource/gray.png</normaloff>:/icons/resource/gray.png</iconset>
   </property>
   <property name="text">
    <string>Invisible</string>
   </property>
  </action>
  <action name="actionSign_out">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resource/gray.png</normaloff>:/icons/resource/gray.png</iconset>
   </property>
   <property name="text">
    <string>Sign out</string>
   </property>
  </action>
  <action name="actionAway">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resource/orange.png</normaloff>:/icons/resource/orange.png</iconset>
   </property>
   <property name="text">
    <string>Away</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>statusAvatarWidget</class>
   <extends>QPushButton</extends>
   <header>statusAvatarWidget.h</header>
  </customwidget>
  <customwidget>
   <class>statusTextWidget</class>
   <extends>QWidget</extends>
   <header>statusTextWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>statusToolButton</class>
   <extends>QToolButton</extends>
   <header>statusToolButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
